assets/src/modules/config/Theme.js

```diff
@@ -0,0 +1,257 @@
+import { BaseObjectConfig } from './Base.js';
+import { ValidationError } from './../Errors.js';
+
+const layerThemeRequiredProperties = {
+    'style': {type: 'sting'},
```

Comment:
`string`

Reply:
TypeScript is used?

Reply:
@nboisteault no typescript only javascript.